.env.template

```diff
@@ -4,3 +4,6 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
```

Comment:
You could actually delete SECRET_KEY here and from flask_config.py as you are no longer using `session` cookies anywhere.